### PR TITLE
Update SASM to v3.13.4

### PR DESCRIPTION
**Pull request by TheJanzap**

### CODE REVIEW

File: io.github.dman95.SASM.yml [hunk 1]
```diff
@@ -22,19 +22,11 @@ finish-args:
...
-    post-install:
-      - >
-        awk -i inplace '/<\/component>/ {print "<releases><release version=\"3.12.2\" date=\"2022-02-06\"/></releases>"} 1'
-        ${FLATPAK_DEST}/share/metainfo/io.github.dman95.SASM.metainfo.xml
     sources:
       - type: git
         url: https://github.com/Dman95/SASM.git
-        branch: v3.12.2
-        commit: 4e74afe3460ecd1d9d102a4b4393113d1b001f8f
+        branch: v3.13.4
```
Comment (hfiguiere) [inline io.github.dman95.SASM.yml:28]:
```suggestion
        tag: v3.13.4
```

never use branch

**Comment by hfiguiere** [comment]
the removed patch contained the appstream file that isn't found upstream. hence the build failure.

**Comment by TheJanzap** [comment]
> the removed patch contained the appstream file that isn't found upstream. hence the build failure.

Yeah, I've seen that. I'm currently trying to add the appstream as a seperate file like in other repos, but I'm not sure how to reference it, since I'm still getting "appstream-missing-appinfo-file" from the linter.

**Comment by hfiguiere** [comment]
I really can't see but my guess is that it is not installed.

> as a seperate file like in other repos 

actually it should be upstreamed. It's not flatpak specific.

**Comment by hfiguiere** [comment]
(but having it here in the mean time is ok)

File: io.github.dman95.SASM.desktop
Comment (hfiguiere) [inline io.github.dman95.SASM.desktop:1]: this is upstream

https://github.com/Dman95/SASM/tree/v3.14.0/Linux/share/applications

No reason to have it here.

File: io.github.dman95.SASM.png
Comment (hfiguiere) [inline io.github.dman95.SASM.png:1]: This is upstream, no reason to have it here 

https://github.com/Dman95/SASM/blob/v3.14.0/images/smallIcon.png

**Comment by TheJanzap** [comment]
Hi @hfiguiere , I don't want to stress you out, but if everything is alright with my commits it would be cool if this could be merged. Thank you for your support :)

**Comment by hfiguiere** [comment]
I am not the maintainer of the  package.

**Comment by hfiguiere** [comment]
bot, build

File: io.github.dman95.SASM.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  
+  <description>
+    <p>
+      SASM (SimpleASM) - simple Open Source cross-platform IDE for NASM, MASM, GAS, FASM assembly languages. SASM has syntax highlighting and debugger. The program works out of the box and is great for beginners to learn assembly language. SASM is translated into Russian, English, Turkish (thanks Ali Goren), Chinese (thanks Ahmed Zetao Yang), German (thanks Sebastian Fischer), Italian (thanks Carlo Dapor), Polish (thanks Krzysztof Rossa), Hebrew (thanks Elian Kamal), Spanish (thanks Mariano Cordoba). Licensed under the GNU GPL v3.0. Based on the Qt.
+    </p>
+  </description>
+  
+  <description xml:lang="ru">
```
Comment (hfiguiere) [inline io.github.dman95.SASM.metainfo.xml:18]: this doesn't validate anymore

File: io.github.dman95.SASM.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  <url type="help">https://github.com/Dman95/SASM/issues</url>
+  <launchable type="desktop-id">io.github.dman95.SASM.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image xml:lang="en">https://dman95.github.io/SASM/images/Screenshot3.png</image>
+    </screenshot>
+    <screenshot>
+      <image xml:lang="en">https://dman95.github.io/SASM/images/Screenshot4.png</image>
+    </screenshot>
+    <screenshot type="default">
```
Comment (hfiguiere) [inline io.github.dman95.SASM.metainfo.xml:34]: there should be only one "default" screenshot

File: io.github.dman95.SASM.metainfo.xml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      SASM (SimpleASM) - простая кроссплатформенная среда разработки для языков ассемблера NASM, MASM, GAS, FASM с подсветкой синтаксиса и отладчиком. В SASM Вы можете легко разрабатывать и выполнять программы, написанные на языке ассемблера. Вводите код в форму и запускайте приложение. Программа работает "из коробки" и хорошо подойдет для начинающих изучать язык ассемблера. Основана на Qt. Распространяется по свободной лицензии GNU GPL v3.0.
+    </p>
+  </description>
+  
+  <url type="homepage">https://dman95.github.io</url>
+  <url type="help">https://github.com/Dman95/SASM/issues</url>
+  <launchable type="desktop-id">io.github.dman95.SASM.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image xml:lang="en">https://dman95.github.io/SASM/images/Screenshot3.png</image>
```
Comment (hfiguiere) [inline io.github.dman95.SASM.metainfo.xml:29]: this shouldn't have any lang, and the localized version should be next to it inside the screenshot tag.

**Pull request merged**